validation/jinja2/validation/search.html

```diff
@@ -25,49 +25,44 @@
 
 {% block content %}
 <h2> Results for <q>{{ search_term }}</q> </h2>
+    {%  for phrase in phrases %}
+    <div class="card segment-card">
```

Comment:
[`<article>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article) may be more appropriate here:

```suggestion
    <article class="card segment-card">
```

Each card is its own self-contained entry, correct?